.github/workflows/pr-differences-mutants.yml

```diff
@@ -9,16 +9,43 @@ on:
       - ready_for_review
     paths:
       - '**.rs'
+  workflow_dispatch:
+    inputs:
+      ignore_timeout:
+        description: "Ignore mutants timeout limit"
+        required: false
+        type: choice
+        options:
+          - true
```

Comment:
Instead of a choice with just one option, I think you can use `type: boolean`, or if you stick with `choice` I think you need to add a false option.

Reply:
good catch - i don't recall seeing boolean as an option in the past, but sure enough it is one:
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#inputs-context

Reply:
If I enable the false option (by adding it or changing the type to boolean), developers outside the @stacks-network/blockchain-team who have write access to create PRs would be able to trigger CI flows lasting less than 6 hours using the `false` input. While I don't expect this to have a major impact, since they can already initiate CI mutants workflows through their PRs, I wanted to double check with you if this behavior is intended.